spec/vim/plugin/syntax_test.vim

```diff
@@ -2,7 +2,7 @@
 let s:debug = 0
 
 function! s:CursorHasGroup(group) abort
-  return synIDattr(synID(line('.'), col('.'), 0), 'name') =~ a:group
+  return synIDattr(synID(line('.'), col('.'), 1), 'name') =~ a:group
```

Comment:
Locally, this passes just fine and makes the check go through transparent groups. That way, the assertion can check the syntax group that is actually *seen*. All the tests seem to be passing, so I'm thinking we weren't relying on it. But I can also put it back to 0 and make the test assertion for `rubyNestedParentheses` instead of for `rubySymbol`. 

Ideally, maybe it should be an option, but it could also be added later when needed.

Reply:
LGTM, thanks.